examples/hosts/app-integration/external-data/src/view/recoilState.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { atom } from "recoil";
+
+export const fetchingExternalData = atom({
```

Comment:
We should document what each of these pieces of state represent in the context of the application.